Add USE_EXTERNAL_APPS permission (#6913)
hammerandchisel/discord-api-docs: docs/topics/Permissions.md

| USE_EXTERNAL_SOUNDS                    | `0x0000200000000000` `(1 << 45)` | Allows the usage of custom soundboard sounds from other servers                                                                                                                                                                                            | V            |
| SEND_VOICE_MESSAGES                    | `0x0000400000000000` `(1 << 46)` | Allows sending voice messages                                                                                                                                                                                                                              | T, V, S      |
| SEND_POLLS                             | `0x0002000000000000` `(1 << 49)` | Allows sending polls                                                                                                                                                                                                                                       | T, V, S      |

> | USE_EXTERNAL_APPS                      | `0x0002000000000000` `(1 << 50)` | Allows using user installed apps not installed to the server. When disabled, users will still be allowed to use their apps but the responses will be ephemeral.                                                                                            | T, V, S      |

Reviewer: ```suggestion
| USE_EXTERNAL_APPS                      | `0x0004000000000000` `(1 << 50)` | Allows using user installed apps not installed to the server. When disabled, users will still be allowed to use their apps but the responses will be ephemeral.                                                                                            | T, V, S      |
```
hex

Author: ty

> | USE_EXTERNAL_APPS                      | `0x0004000000000000` `(1 << 50)` | Allows using user installed apps not installed to the server. When disabled, users will still be allowed to use their apps but the responses will be ephemeral.                                                                                            | T, V, S      |

Reviewer 2: ```suggestion
| USE_EXTERNAL_APPS                      | `0x0004000000000000` `(1 << 50)` | Allows the use of user-installed apps that are not already installed to the server. When disabled, users will still be able to use them, but the responses will be forced ephemeral.                                                                                            | T, V, S      |
```

> | USE_EXTERNAL_APPS                      | `0x0004000000000000` `(1 << 50)` | Allows using user installed apps not installed to the server. When disabled, users will still be allowed to use their apps but the responses will be ephemeral.                                                                                            | T, V, S      |

Reviewer 3: ```suggestion
| USE_EXTERNAL_APPS                      | `0x0004000000000000` `(1 << 50)` | Allows user-installed apps not installed to the server to send public responses. When disabled, users will still be allowed to use their apps but the responses will be ephemeral.                                                                                            | T, V, S      |
```

Reviewer 3: maybe something like this that focuses on the responses? because right now the first sentence almost feels misleading in context.

Author: Made an update, I really like your suggestion.
